geowebcache/core/src/main/java/org/geowebcache/config/XMLConfiguration.java

```diff
@@ -135,6 +135,13 @@
     private String templateLocation;
 
     private GridSetBroker gridSetBroker;
+    
+    /**
+     * A flag for whether the config needs to be loaded at {@link #initialize(GridSetBroker)}. If
+     * the constructor loads the configuration, will set it to false, then each call to initialize()
+     * will reset this flag to true
+     */
+    private boolean reloadConfigOnInit = true;
```

Comment:
So this is to indicate that the config file was read earlier than normal so that when it reaches the normal point to read it, it won't be read twice?